packages/calypso-e2e/test/browser-helper.test.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file Tests the BrowserHelper functions using process.env and stubbed config file.
+ * @author Edwin Takahashi
```

Comment:
We usually don't have these header in files. It doesn't add more info to what `git blame` tells us.

Reply:
Will be addressed in next patch, and the removal will include another instance of this header.